--- krusader/Panel/krdetailedview.cpp	2004-10-24 18:17:31.000000000 +0200
+++ krusader/Panel/krdetailedview.cpp	2004-10-31 16:50:49.000000000 +0100
@@ -1034,8 +1034,8 @@
       if ( item == startItem )
          return ;
       if ( caseSensitive ? item->name().startsWith( str ) : item->name().lower().startsWith( str.lower() ) ) {
-         makeItemVisible( item );
          setCurrentItem( item->name() );
+			makeItemVisible( item );
          return ;
       }
    }
--- krusader/Panel/panelfunc.cpp	2004-10-20 02:03:26.000000000 +0200
+++ krusader/Panel/panelfunc.cpp	2004-11-11 21:40:09.000000000 +0100
@@ -650,6 +650,11 @@
 	if ( PackGUI::type == QString::null )
 		return ; // the user canceled

+	// check for partial URLs
+	if( !PackGUI::destination.contains(":/") && !PackGUI::destination.startsWith("/")  ){
+		PackGUI::destination = panel->virtualPath+"/"+PackGUI::destination;
+	}
+
 	bool packToOtherPanel = ( PackGUI::destination == panel->otherPanel->virtualPath );

 	// on remote URL-s first pack into a temp file then copy to its right place
@@ -779,6 +784,10 @@
 	if ( dest == QString::null )
 		return ; // the usr canceled

+	// check for partial URLs
+	if( !dest.contains(":/") && !dest.startsWith("/")  ){
+		dest = panel->virtualPath+"/"+dest;
+	}
 	bool packToOtherPanel = ( dest == panel->otherPanel->virtualPath );

 	for ( unsigned int i = 0; i < fileNames.count(); ++i ) {
--- krusader/Panel/krdetailedviewitem.cpp	2004-10-23 00:01:44.000000000 +0200
+++ krusader/Panel/krdetailedviewitem.cpp	2004-11-01 09:20:40.000000000 +0100
@@ -74,8 +74,8 @@
     // set text in columns, according to what columns are available
     int id = KrDetailedView::Unused;
     if ((id = mimeColumn) != -1) {
-      QString tmp = _vf->vfile_getMime();
-      setText(id, tmp.mid(tmp.find('/')+1));
+      QString tmp = KMimeType::mimeType(_vf->vfile_getMime())->comment();
+      setText( id, tmp );
     }
     if ((id = sizeColumn) != -1) {
       if (_vf->vfile_isDir() && _vf->vfile_getSize() <= 0) setText(id, "<DIR>");
--- krusader/UserAction/useractionxml.cpp	2004-10-12 16:20:43.000000000 +0200
+++ krusader/UserAction/useractionxml.cpp	2004-11-03 20:51:12.000000000 +0100
@@ -58,14 +58,17 @@
     }
     file.close();
     
-    QDomElement root = _doc->documentElement();
-    // check if the file got the right root-element (ACTION_ROOT) - this finds out if the xml-file read to the DOM is realy an krusader useraction-file
-    if( root.tagName() != ACTION_ROOT ) {
-      kdWarning() << "UserActions: the actionfile's root-element isn't called "ACTION_ROOT", using " << _filename << endl;
-      delete _doc; _doc = 0;
+    if ( _doc ) {
+      QDomElement root = _doc->documentElement();
+      // check if the file got the right root-element (ACTION_ROOT) - this finds out if the xml-file read to the DOM is realy an krusader useraction-file
+      if( root.tagName() != ACTION_ROOT ) {
+        kdWarning() << "UserActions: the actionfile's root-element isn't called "ACTION_ROOT", using " << _filename << endl;
+        delete _doc; _doc = 0;
+      }
     }
-      
-  } else
+
+  } // if ( file.open( IO_ReadOnly ) )
+  else
     _doc->appendChild( _doc->createElement( ACTION_ROOT ) ); // create new actionfile by adding a root-element ACTION_ROOT
     
 }
