--- krusader/krusader.orig	2004-11-01 11:58:28.000000000 +0100
+++ krusader/krusader.desktop	2004-11-01 12:05:24.211643696 +0100
@@ -4,13 +4,14 @@
 Exec=krusader -caption "%c" %i %m  
 Icon=krusader.png
 MiniIcon=krusader.png
-Terminal=0
+Terminal=false
 Type=Application
 DocPath=krusader/index.html
-Categories=Qt;KDE;Utility;
+Categories=Qt;KDE;Utility;FileManager;
 #Categories=Qt;KDE;System;
 Comment=Twin-Panel File Manager
 Comment[hu]=Kétpaneles fájlkezelő
+Comment[pl]=Dwupanelowy zarządca plików
 GenericName=File Manager
 GenericName[af]=Lêer Bestuurder
 GenericName[ar]=مدير الملفات
@@ -84,3 +85,4 @@
 GenericName[zh_CN]=文件管理器
 GenericName[zh_TW]=檔案管理程式
 GenericName[zu]=imenenja yamafayela
+# vi: encoding=utf-8
