--- krusader/MountMan/kmountman.cpp.orig	2004-10-20 00:46:42.000000000 +0200
+++ krusader/MountMan/kmountman.cpp	2004-10-22 14:24:23.398732536 +0200
@@ -62,7 +62,7 @@
 	}
 
 	// list of FS that we don't manage at all
-	invalid_fs << "swap" << "/dev/pts" << "tmpfs" << "devpts" << "sysfs" << "rpc_pipefs" << "usbfs" << "binfmt_misc";
+	invalid_fs << "swap" << "/dev/pts" << "tmpfs" << "devpts" << "sysfs" << "rpc_pipefs" << "usbfs" << "binfmt_misc" << "devfs" << "usbdevfs" << "selinuxfs";
 #if defined(BSD)
 	invalid_fs << "procfs";
 #else
