--- krusader-1.30/doc/en/index.docbook.orig	Sat Nov 15 11:42:14 2003
+++ krusader-1.30/doc/en/index.docbook	Fri Dec  5 16:21:43 2003
@@ -1,6 +1,7 @@
 <?xml version="1.0" ?>
 <!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1-Based Variant V1.0//EN" "dtd/kdex.dtd" [
   <!ENTITY krusader '<application>Krusader</application>'>
+  <!ENTITY package 'krusader'>
   <!ENTITY kapp "&krusader;"><!-- replace test here -->
   <!ENTITY % addindex "IGNORE">
   <!ENTITY % English "INCLUDE"><!-- change language only here -->
