--- krusader-1.60.0-beta1/krusader/krusader_root-mode.desktop	2005-01-13 00:10:48.000000000 +0100
+++ krusader-1.60.0-beta1/krusader/krusader_root-mode.desktop	2005-03-04 00:57:38.061467864 +0100
@@ -3,6 +3,7 @@
 Name=Krusader - root-mode
 Name[de]=Krusader - root-Modus
 Name[cs]=Krusader s právy roota
+Name[pl]=Krusader - tryb administratora
 Exec=krusader -caption "%c" %i %m  
 Icon=krusader_red.png
 Terminal=false
@@ -15,6 +16,7 @@
 Comment=Twin-Panel File Manager with root-privileges
 Comment[de]=Zwei-Fenster Dateimanager mit root-Privilegien
 Comment[cs]=Dvojpanelový správce souborů s právy Roota
+Comment[pl]=Dwupanelowy zarządca plików - tryb administratora
 GenericName=File Manager
 GenericName[af]=Lêer Bestuurder
 GenericName[ar]=مدير الملفات
